man/makereducedfrompartition.Rd

```diff
@@ -0,0 +1,44 @@
+\name{make_reduced_from_partition}
+\alias{make_reduced_from_partition}
+\title{Output reduced adjacency matrices given a partitioning of the nodes into appropriate groups}
+\description{
+Takes an adjacency matrix and a partition of the nodes, and returns a
+reduced adjacency matrix and the corresponding cutoff density/degree. The 
+\pkg{sna} package must be installed, but does not need to be attached.
+}
+\usage{
+make_reduced_from_partition(adj_mat, partition, stat = 'density')
+}
+\arguments{
+  \item{adj_mat}{A adjacency matrix}
+  \item{partition}{A vector that gives the desired partitioning of the nodes. }
+  \item{stat}{The statistic that will be used to determine if edges should be included in a reduced network.  Default value is \code{density}}
```

Comment:
Note that the other implemented option is `degree`?

Reply:
agree